system/Format/JSONFormatter.php

```diff
@@ -56,7 +56,8 @@ class JSONFormatter implements FormatterInterface
 	 */
 	public function format($data)
 	{
-		$options = JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES | JSON_PARTIAL_OUTPUT_ON_ERROR;
+		$options = $config->formatterOptions['application/json'] ?? 0;
```

Comment:
```suggestion
		$options = $config->formatterOptions['application/json'] ?? JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES;
```
This way we can introduce this change with new configuration variable really easy.